app/Channels.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Channels extends Model
+{
+    protected $fillable = [
+        'user_id',
+        'fcm_token'
+    ];
+
+    public function user()
+    {
+        return $this->hasOne('App\User');
```

Comment:
```suggestion
        return $this->hasOne(User::class);
```

Usar a sugestão acima, incluindo `use App\User` no começo da classe. Em qualquer lugar que o nome de uma classe for necessária, usar `Nome::class` ao invés de uma string.